<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_PathFinder</class>
 <widget class="QMainWindow" name="C_PathFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>723</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>723</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Path Finder Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gv_Map">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>651</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CenterMap">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveRightMap">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveLeftMap">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveUpMap">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomInMap">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveDownMap">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomDefaultMap">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomOutMap">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>World map</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkB_FollowRobot">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Robot Tracking</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamPos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamPos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_nextPos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_nextPos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_frameRate">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_frameRate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkB_DynamicMargin">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>680</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dynamic Margin</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
